### ISSUE TITLE
Restyling the Code of Honor

### ISSUE THREAD

**Issue by pedro16v** [catalpainternational/canoe#591]
(I could swear I had already opened this issue)

Due to recent changes on the Canoe DM, the warning that users see before taking an exam is now a roll-up instead of a card.

The current design shows alternative text but I've pinged @andrewwapling on InVision to provide a new look at it.

**Comment by pedro16v** [comment]
This is the [current state](https://projects.invisionapp.com/share/FQ10IGYR7N3Y#/screens). We're still waiting on review and approval. 

<img width="321" alt="Screenshot 2021-04-09 at 12 10 50" src="https://user-images.githubusercontent.com/2813677/114123208-d240b980-992c-11eb-9a6c-797a29b6753b.png">

**Comment by pedro16v** [comment]
Here's the latest

![Screenshot 2021-04-14 at 15 01 30](https://user-images.githubusercontent.com/2813677/114661706-65fdf580-9d32-11eb-9fd8-9c5de156f18a.png)

**Comment by andrewwapling** [comment]
Hi @pedro16v yes I believe this was located in another issue. So that we can capture it here as well I've added the direct link to the prototype screen below: https://invis.io/F21135B8WXH8

Hope this helps, Andy
